shell/plugins/steve/subscribe.js

```diff
@@ -456,42 +451,58 @@ export const actions = {
   }, e) {
     clearTimeout(state.queueTimer);
     state.queueTimer = null;
-    if (e.type === EVENT_DISCONNECT_ERROR) {
-      // determine if websocket notifications are disabled
-      const perfSetting = rootGetters['management/byId'](MANAGEMENT.SETTING, SETTING.UI_PERFORMANCE);
-      let disableGrowl = false;
+
+    // determine if websocket notifications are disabled
+    const perfSetting = rootGetters['management/byId'](MANAGEMENT.SETTING, SETTING.UI_PERFORMANCE);
+    let disableGrowl = false;
+
+    if (!disableGrowl) {
```

Comment:
We should use this guard on the success growl above as well

Reply:
done